tests/utils/profilehandler/interface.go

```diff
@@ -0,0 +1,48 @@
+package profilehandler
+
+type Profile struct {
+	Name              string             `yaml:"as,omitempty"`
+	Version           string             `yaml:"version,omitempty"`
+	ChannelGroup      string             `yaml:"channel_group,omitempty"`
+	Region            string             `yaml:"region,omitempty"`
+	ClusterConfig     *ClusterConfig     `yaml:"cluster,omitempty"`
+	AccountRoleConfig *AccountRoleConfig `yaml:"account-role,omitempty"`
```

Comment:
Seems like we need to add a `Prefix` for the cluster name, the account-roles, the operator-roles, and so on. And maybe we need another global environment to allow customized.

Reply:
For now we will generate automatically. In future, if necessary we will expose it.